src/lib/command-utils.ts

```diff
@@ -28,3 +28,18 @@ export async function getSwankyConfig(): Promise<SwankyConfig> {
     throw new Error("No 'swanky.config.json' detected in current folder!");
   }
 }
+
+export function resolveNetworkUrl(
+  config: SwankyConfig,
+  networkName: string
```

Comment:
sorry, missed this before.
Why not make network name optional instead of empty string?
That way you can return default on `!networkName`, or even better use the type guard, and avoid the `??` check on calling it.

```suggestion
  export function resolveNetworkUrl(
  config: SwankyConfig,
  networkName?: string
  ): string {
  if (typeof networkName === "undefined") return DEFAULT_NETWORK_URL
  ...
  }
```